tests/index/elastic/v7/test_index_get_del.py

```diff
@@ -0,0 +1,232 @@
+import numpy as np
+import pytest
+
+from docarray import DocumentArray
+from docarray.index import ElasticDocumentIndex
+from tests.index.elastic.fixture import start_storage_v7  # noqa: F401
+from tests.index.elastic.fixture import DeepNestedDoc, FlatDoc, NestedDoc, SimpleDoc
+
+
```

Comment:
It is probably best to do this in (a) separte file(s), but can we add tests for:
- persisting data and loading it with a new instance of our index class
- column configurations, where the user for example changes the DB column type of one field (from `keyword` to `text` or something like that)
- a test where we index an `ImageDoc` with some image tensor, make sure that that can be handled properly
- tensorflow and torch support (after you rebase this should work automatically, but good to test anyways)
- maybe other stuff that I am forgetting

This is my fault, I didn't write all necessary tests on the HNSLib implementation. But unfortunately I think we need more, I am also currently working on this for HNSWLib :)